flow/record/stream.py

```diff
@@ -17,6 +18,9 @@
 
 log = logging.getLogger(__package__)
 
+aRepr = reprlib.Repr()
+aRepr.maxother = 255
```

Comment:
```suggestion
aRepr = reprlib.Repr(maxother=255)
```
?

Reply:
Sadly Repr() doesn't have a initializer for this:

```python
Init signature: reprlib.Repr()
Docstring:      <no docstring>
Source:
class Repr:

    def __init__(self):
        self.maxlevel = 6
        self.maxtuple = 6
        self.maxlist = 6
        self.maxarray = 5
        self.maxdict = 4
        self.maxset = 6
        self.maxfrozenset = 6
        self.maxdeque = 6
        self.maxstring = 30
        self.maxlong = 40
        self.maxother = 30
```

Reply:
Oh I missed this line in the documentation:
> Changed in version 3.12: Allow attributes to be set via keyword arguments.